openid-4-verifiable-presentations-1_0.md

```diff
@@ -559,11 +565,276 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+# Verifiable Presentation Query Language {#vp_query}
+
+The Verifiable Presentation Query Language (VP Query) is a JSON-encoded query
```

Comment:
Same comment as above r.e naming the VP isn't the subject of the query IMO the credential is.

Reply:
`Verifiable Credential Query Language (VC Query)` would work

Reply:
digital credential?

Reply:
The whole spec talks about "verifiable credentials", so why change it here?

Reply:
I can save my comment for a broader issue - it's a confusing term, and doubly confusing since the group is named digital credentials